src/app/partials/nav/header-static/header-static.component.scss

```diff
@@ -3,19 +3,299 @@
 header{
 	position: fixed;
 	left: 0;
-	width: 100%;
 	top: 0;
 	color: white;
-	background-color: rgba(255,255,255,0);
 	width: 100%;
-	padding: 10px 40px;
 	box-sizing: border-box;
 	z-index: 1000;
-	font-size: $fs-14;
 	transition: all 0.2s ease-in-out;
 	.logo{
 		img{
 			width: 30px;
 		}
 	}
 }
+
+@mixin fire($inverse: false) {
+    @if $inverse {
+        background: linear-gradient(-90deg, $orange, $yellow);
+    } @else {
+        background: linear-gradient(-90deg, $yellow, $orange);
+    }
+}
+@mixin shadow-btn {
+	box-shadow: 0px 0px 5px $overlay-light;	
+}
+@mixin shadow-btn-large {
+	box-shadow: 0px 15px 20px $overlay-light;
+}
+@mixin shadow-header-large {
+	box-shadow: 0px 0px 5px $overlay-light;	
+}
+
+.dark-overlay{
+	width:100%;
+	height:100%;
+	position:fixed;
+	z-index:100;
+	background:$overlay-light;
+	overflow:hidden;
+}
+
+.menu-expander{
+	display:none;
+	color: black;
+	cursor:pointer;
+	position:absolute;
+    top:25px;
+    right:10px;
+    z-index:100;
+
+}
+
+.menu-bar1, .menu-bar2, .menu-bar3 {
+    width: 30px;
+    height: 5px;
+    
+    margin: 6px 0;
+    transition: 0.4s;
+}
+
+.dark-bg {
+	background: black;
+}
+
+.white-bg {
+	background: white;
+}
+
+.menu-change {
+	.menu-bar1 {
+	   -webkit-transform: rotate(-45deg) translate(-9px, 6px);
+    	transform: rotate(-45deg) translate(-9px, 6px);
+    }
+}
+
+.menu-change {
+	.menu-bar2 {
+		opacity: 0;
+	}
+}
+
+.menu-change {
+	.menu-bar3 {
+	    -webkit-transform: rotate(45deg) translate(-8px, -8px);
+    	transform: rotate(45deg) translate(-8px, -8px);
+    }
+}
+
+.header-shadow{
+	@include shadow-btn;
+}
+
+.header-shadow-large{
+	@include shadow-header-large;
+}
+
+.dark-text{
+	color:$gray-darker !important;
+}
+
+.invert-color{
+	filter: invert(100%);
+}
+
+.logo-container {
+	text-align: right;
+	display:inline-block;
+	float: left;
+	width:10%;
+	margin-top:30px;
+	margin-left:30px;
+	img:hover {
+		cursor:pointer;
+	}
+	img {
+		height:40px;
+	}
+}
+
+.navlinks-container {
+	display:inline-block;
+	float: right;
+	width:70%;
+	position:relative;
+	text-align:right;
+	transition:height 0.5s ease;
+	padding-bottom:10px;
+	padding-right:10px;
+	div{
```

Comment:
Give div a class, otherwise it will apply styles to every div.  @live-wire

Reply:
👍